circle.yml

```diff
@@ -11,7 +11,7 @@ machine:
 dependencies:
   override:
     - docker info
-    - docker build -t=$PRIVATE_REGISTRY/$CIRCLE_PROJECT_REPONAME:b$CIRCLE_BUILD_NUM .
+    - docker build --rm -t=$PRIVATE_REGISTRY/$CIRCLE_PROJECT_REPONAME:b$CIRCLE_BUILD_NUM .
```

Comment:
FYI: `--rm` doesn't work on Circle because we don't have permissions to remove layers, it'll just spit warnings to stderr. Also, since 1.9, `--rm` is the default.